homeassistant/components/xiaomi_miio/strings.json

```diff
@@ -276,16 +276,16 @@
         "name": "Total dust collection count"
       },
       "main_brush_left": {
-        "name": "Main brush left"
+        "name": "[%key:component::roborock::entity::sensor::main_brush_time_left::name%]"
       },
       "side_brush_left": {
-        "name": "Side brush left"
+        "name": "[%key:component::roborock::entity::sensor::side_brush_time_left::name%]"
       },
       "filter_left": {
-        "name": "Filter left"
+        "name": "[%key:component::roborock::entity::sensor::filter_time_left::name%]"
       },
       "sensor_dirty_left": {
-        "name": "Sensor dirty left"
```

Comment:
This could be misinterpreted (like it was in German Lokalise translation where the meaning now is "left sensor is dirty"), it should represent the time left until you need to clean the sensor. That's why I thought better use the existing translations of Roborock.